packages/SwingSet/src/liveslots/liveslots.js

```diff
@@ -863,6 +880,7 @@ function build(
     lsdebug(
       `ls.qm send(${JSON.stringify(targetSlot)}, ${legibilizeMethod(prop)}) -> ${resultVPID}`,
     );
+    assertAcceptableSyscallCapdataSize(serMethargs);
```

Comment:
I think we should do this check immediately after the `m.serialize()` above, because if we make it this far and then bail, we'll call `retainExportedVref` and create a new (tracked) result promise and VPID, all of which will be leaked. If we bail earlier, we avoid that leak.